README.md

```diff
@@ -1,29 +1,38 @@
 
-# Blocking POC 
+# Anonlink Client
 
-A simple end to end demonstrator of blocking for record linkage.
+Client-facing API to interact with anonlink system including command line tools and Rest API communication.
+Anonlink system needs the following three components to work together:
 
-Output should be candidate blocks.
+* [clkhash](https://github.com/data61/clkhash)
+* [blocklib](https://github.com/data61/blocklib)
+* [anonlink-entity-service](https://github.com/data61/anonlink-entity-service)
 
+This package provides an easy to use API to interact with the above packages to complete a record linkage job.
+
+The way to interact with anonlink system is via Command Line Tool `anonlink`. You can hash data containing PII (Personal
+ Identifying Information) locally using `anonlink hash`, generate candidate blocks locally to scale up record linkage 
+ using `anonlink block`, create a record linkage job in entity service with `anonlink create-project` etc.
+
+### Installation
+Currently manual install:
+
+```python3
+pip install git+https://https://github.com/data61/clkhash
 ```
-{
-  'records': [clks],
-  'blocks': {
-      'block id 1': [record ids]
-  }
-}
-```
 
-Two methods have been implemented that map to a list of records.
+### Documentation
+https://clkhash.readthedocs.io/en/stable/cli.html
+Note that the documentation are for clkhash now, we will add a readthedocs page for anonlink-client very soon.
+
+### CLI Tool
+After installation, you should have a `anonlink` program in your path. For
+example, to hash PII data  `alice.csv` locally with schema `schema.json` and secret `horse`, run:
+```bash
+$ anonlink hash 'alice.csv' 'horse' 'schema.json' 'clk.json'
```

Comment:
Perhaps a good time to rename `hash` to `encode`? @wilko77

Reply:
I wouldn't use the filename `clk.json` either, perhaps `encoded-entities.json`

Reply:
An issue is created for the renaming proposal #15